fstar-helpers/core-models/proofs/fstar/extraction/Makefile

```diff
@@ -0,0 +1,270 @@
+# This is a generically useful Makefile for F* that is self-contained
```

Comment:
Why is this file not pointing to Makefile.generic?

Reply:
Ah yes, thanks, I'm going to fix that.

Reply:
Done